Doc/library/importlib.metadata.rst

```diff
@@ -111,12 +118,31 @@ This package provides the following functionality via its public API.
 Entry points
 ------------
 
-The ``entry_points()`` function returns a collection of entry points.
+.. class:: EntryPoint
+
+    Details of an installed entry point (as described below).
+
+.. class:: EntryPoints
+
+    Details of a collection of installed entry points (as described below).
```

Comment:
@ncoghlan I think it's fine to leave off "as described below".

```suggestion
    Details of a collection of installed entry points.
```

Reply:
Applied locally.